HHS/Head-Start-TTADP
# [TTAHUB-2977] Add first version of CDI grant filter

## Description of change

This will be a bit scaled back from what the ticket initially asked for as it overlaps with the work Jon is currently doing for grants. And the QA work that Garrett is completing for the SSDI.

For now this filter will be purely on the CDI column on the grants table (no associated grants). It will also be currently excluded from the QA dashboard as the page is still a WIP. 

**Note:** When filtering inactive grants on the RTR search page we by default limit the possible inactive grants using the below query. You will need to account for this when matching to what's in the db via a query.
`("grants"."endDate" > '2020-08-31 04:00:00.000 +00:00' AND ("grants"."inactivationDate" IS NULL 
OR "grants"."inactivationDate" > '2020-08-31 04:00:00.000 +00:00'))`

## How to test

- Review the code.
- Make sure the grant filter works as expected on the various pages (see ticket)

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2977


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
